### DISCUSSION TITLE
What is this .md flavour?

### DISCUSSION THREAD

**Discussion by princesslunaequestrian** [thelfer/tfel#475] [Q&A] [answered]
The SOVS page on the web is down for some reason, so I went to the github docs.

However, the .md file contains latex, md and html syntax all in one.

How should I view it properly?

[[docs/web/sovs.md](url)](https://github.com/thelfer/tfel/blob/master/docs/web/sovs.md)

**Comment by thelfer** [comment, marked as answer]
Dear @princesslunaequestrian,

The SOSV page is not down. The sourceforge site has been closed (it has not been updated for years) and moved here: https://thelfer.github.io/tfel/web/sovs.html

Concerning your question, the markdown flavour is [pandoc](https://pandoc.org/MANUAL.html)'s one, detailed here: https://pandoc.org/MANUAL.html#pandocs-markdown

**Reply by princesslunaequestrian** [reply]
Thank you for the answer!
